src/ngraph/runtime/gpu/cudnn_emitter.cpp

```diff
@@ -13,12 +13,16 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *******************************************************************************/
+#include <algorithm>
 #include <iostream>
+#include <sstream>
 #include <vector>
 
 #include "ngraph/runtime/gpu/cudnn_emitter.hpp"
 #include "ngraph/runtime/gpu/gpu_primitive_emitter.hpp"
 #include "ngraph/runtime/gpu/gpu_runtime_context.hpp"
+#include "ngraph/runtime/gpu/gpu_util.hpp"
+#include "ngraph/util.hpp"
 
 using namespace ngraph;
 using namespace ngraph::runtime::gpu;
```

Comment:
I know it is not part of your code, but we should not use `using namespace` beyond the first namespace, so `using namespace ngraph;` is ok but `using namespace ngraph::runtime::gpu;` is not

Reply:
I will change these, but if it is ok, I would prefer to make a separate PR to remove them because we do this in many places and it might obfuscate this PR. I also notice this in the the IA transformer, e.g. mkldnn_emitter.cpp